03/app.js

```diff
@@ -0,0 +1,10 @@
+const btns = document.querySelectorAll('button');
+
+const changeName = function (e) {
+    e.target.textContent = 'clicked';
+    console.log(e.target.textContent);
```

Comment:
Nie do końca działa zgodnie z opisem zadania.
Drugi i kolejne kliknięcie powoduje wyświetlenie komunikatu w konsoli.
Spróbuj tak zmodyfikować to zadania, aby drugi i kolejne kliknięcie nie wywoływało funkcji `changeName`